website/docs/docs/cloud/connect-data-platform/connect-snowflake.md

```diff
@@ -68,3 +73,18 @@ more information on configuring a Snowflake OAuth connection in dbt Cloud, pleas
 ## Configuration
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [Snowflake-specific configuration](/reference/resource-configs/snowflake-configs).
+
+## Troubleshooting
+<!--might need to turn this into details toggle if more troubleshooting items arise -->
+
+If you're receiving a `Could not deserialize key data` or `JWT token` error, refer to the following causes and solutions:
+
+- **Error: `Could not deserialize key data`**
+  - Possible causes &mdash; This could be because of mistakes like not copying correctly, missing dashes, or leaving out commented lines.
+  - Solution &mdash; You can either manually type the key values or copy the key from its source and paste it into a text editor to verify it before using it in dbt Cloud. 
```

Comment:
I don't think that anyone will type hundreds/thousands of characters 1 by 1 😄 , so I might remove this suggestion.

Reply:
oh ok, i noticed some of the solutions from support resulted in this [see this zendesk ticket](https://dbtcloud.zendesk.com/agent/tickets/52709), which is why i added it here.